### PR TITLE
Fixes #187.  Change RFG URL http to https. (Ref: https://realfavicong…

### LINKED ISSUE #187: RFG now only uses https
I just noticed that the URL `http://realfavicongenerator.net/api/favicon` now redirects to `https`. Hence, whenever i try to run my gulp script i get, as an error:

```
TypeError: Cannot read property 'result' of undefined
```

Looking at the response, it returned the status code 301.  Changing [line 467](https://github.com/evilebottnawi/favicons/blob/4f6e8bd19115c2b0067e28f9df349905aea2d814/helpers-es5.js#L467) http to https make everything work.

I'll make a PR in awhile.

### PR DESCRIPTION
**Pull request by GheloAce**
…enerator.net/api/non_interactive_api)

### CODE REVIEW

File: helpers-es5.js [hunk 1]
```diff
@@ -496,4 +496,4 @@ var path = require('path'),
     }
 
     module.exports = helpers;
-})();
```
Comment (jantimon) [inline helpers-es5.js]: Can you please remove that change please?

**Comment by zsoldosp** [comment]
@GheloAce @jantimon seeing how this PR is stuck for two weeks and I've run into the same problem and thus keen to have it fixed - would it be OK to make a separate PR (with acknowledgment for the first commit in the commit message) for it so a working version could be released to npmjs? (also, should the version number be also bumped)

**Comment by jrmykolyn** [comment]
Hello! Any movement on this PR? Hoping to pull this update.

**Comment by evilebottnawi** [comment]
Close as invalid. We drop support RFG in new major version.

**Pull request closed without merging**